dbt_copilot_helper/templates/addons/env/rds-postgres.yml

```diff
@@ -46,7 +46,8 @@ Mappings:
       MultiAZ: {{ config.multi_az }}
       AllocatedStorage: {{ config.volume_size }}
       MaxAllocatedStorage: {{ (config.volume_size * 1.26)|round|int }}
-      StorageType: {% if env_name == "prod" %}io1{% else %}gp3{% endif %}
+      StorageType: {% if env_name == "prod" or env_name == "production" %}io1{% else %}gp3{% endif %}
+      Iops: {{ config.iops if config.iops else '""' }}
```

Comment:
Could we set a default value for iops if the env is prod, or update the validation so enforce the iops property for prod.